packages/target-truffle-v5/package.json

```diff
@@ -44,8 +44,10 @@
     "ganache-cli": "^6.9.1",
     "mocha": "^8.2.0",
     "ts-node": "^8.5.4",
+    "typechain": "*",
     "typescript": "3.9.0-beta",
     "web3": "^1.2.6",
-    "typechain": "*"
+    "web3-core": "^1.3.1",
```

Comment:
why do we need these new deps?

Reply:
Because we imported type of `web3-utils` and `web3-core` in `packages/target-truffle-v5/static/types.d.ts`.
https://github.com/ethereum-ts/TypeChain/pull/317/files/5e749a452ab9e602e8f9e773d114c44dddbc0f10#diff-6f45ea93d39575567adc3eb0b242f1ec2a2eda912f9436c5d883111c02fced66